docs/getting-started.md

```diff
@@ -236,7 +235,7 @@ I0324 23:54:00.260254   27739 kind.go:72] Ran: kind [delete cluster --name=clust
 ```
 </details>
 
-The created KIND cluster is ephemeral and is cleaned up automatically when done. During the cluster creation, the KIND configuration is written to a local directory and can be retrieved using `kind get kubeconfig-path --name="clusterapi"`.
+The created KIND cluster is ephemeral and is cleaned up automatically when done. During the cluster creation, the kubectl context is set to "kind-kind" and can be retrieved using `kubectl cluster-info --context kind-kind`.
```

Comment:
should this be 
```suggestion
The created KIND cluster is ephemeral and is cleaned up automatically when done. During the cluster creation, the kubectl context is set to "kind-clusterapi" and can be retrieved using `kubectl cluster-info --context kind-clusterapi`.
```
?

Reply:
just wondering why it's inconsistent with the context in the Makefile

Reply:
you are correct, it should be `kind-clusterapi`, will fix that

Reply:
is there something else you are wondering about in the Makefile?

Reply:
no just which kind context is the right one. Thanks! after that lgtm